docs/indyav/hardware/go_kart/mechanical.rst

```diff
@@ -0,0 +1,4 @@
+Mechanical Documentation
+========================
+Not sure if this is needed or what would go here. Maybe part drawings
```

Comment:
`Not sure if this is needed or what would go here. Maybe part drawings` -> `TODO`